pkg/drivers/kvm/network.go

```diff
@@ -478,7 +478,7 @@ func ifListFromAPI(conn *libvirt.Connect, domain string) ([]libvirt.DomainInterf
 	}
 	defer func() { _ = dom.Free() }()
 
-	ifs, err := dom.ListAllInterfaceAddresses(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE)
+	ifs, err := dom.ListAllInterfaceAddresses(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_ARP)
```

Comment:
how about in case of Error, we try the Old way ? as a fail over

Reply:
Hm..not sure.

There are 2 possibilities(as we have with `DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE`):
* `ListAllinterfaceAddresses` is success and returns nothing(very rare to occur. we should at least have address from **default** network)
* `ListAllinterfaceAddresses` fails to list returning an error

Here's my suggestion. We can fall back in case `DOMAIN_INTERFACE_ADDRESSES_SRC_ARP` fails but log differently in above two scenarios.

Reply:
yeah that sounds like a good idea lets do a fall back and log

Reply:
Done